examples/text-feature-extraction/run_example.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
```

Comment:
Suggestion: File name should be specific like ` run_feature_extraction.py` similar to how the other example scripts are named